src/sentry/static/sentry/app/components/onboardingWizard/toDoItem.tsx

```diff
@@ -84,19 +84,19 @@ class TodoItem extends React.Component<Props, State> {
     switch (task.status) {
       case 'complete':
         description = tct('[user] completed [dateCompleted]', {
-          user: task.user,
+          user: task.user?.name,
```

Comment:
We don't actually use must of the user object yet on the frontend, but I will be using it to display a user avatar soon.